### ISSUE TITLE
vmui/predefined dashboards showLegend parameter from dashboard docs does not work

### ISSUE THREAD

**Issue by kkartaltepe** [VictoriaMetrics/VictoriaMetrics#7565]
### Describe the bug

https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/app/vmui#predefined-dashboards covers the panel settings for predefined dashboards, the documented "showLegend" appears to do nothing.

### To Reproduce

Add `"showLegend": false` to one of the panels from the sample. Full panel settings in my test:

```
{
          "title": "Per-job CPU usage",
          "width": 6,
          "expr": ["sum(rate(process_cpu_seconds_total)) by (job)"],
	 "showLegend": false
        },
```

### Version

`victoria-metrics-20241104-142908-tags-v1.106.0-0-g371e193279`

### Logs

There is no log for using this parameter.

### Screenshots

![screen_20241117210347](https://github.com/user-attachments/assets/17273f02-060e-4af9-84f1-f543923762d9)

Shows the legend displayed on both panels when it should be hidden on the first panel.

**Comment by kkartaltepe** [comment]
This is also the case for the alias property it seems:

```
 {
          "unit": "MB",
          "expr": [
            "sum(vm_data_size_bytes{type!=\"indexdb\"}) / 1024 / 1024",
            "vm_free_disk_space_bytes / 1024 / 1024"
          ],
          "alias": [
            "usage space",
            "free space"
          ]
        }
```

From the docs shows up as "Query 1" instead of "usage space" (it might be useful to document what aliasing means but im assuming its naming the queries).
![screen_20241117211025](https://github.com/user-attachments/assets/0695d946-73e5-4f0e-b0bb-8d3f75ddc354)

**Comment by f41gh7** [comment]
The bugfix was included into [v1.107.0](https://docs.victoriametrics.com/changelog/#v11070) and [v1.102.8](https://docs.victoriametrics.com/changelog/#v11028)
